eagleproject/core/blockchain.py

```diff
@@ -208,11 +231,14 @@ def build_asset_block(symbol, block_number):
                 block_number,
             )
 
+    ora_tok_usd_min = 0 if symbol == 'COMP' else priceUSDMint(VAULT, symbol, block_number)
```

Comment:
Not sure if it's acceptable to use `0` for these Oracle fields, but I needed to do this to satisfy the current schema. We'll want to revisit this later if we start fetching COMP/USD exchange rates as a part of our harvesting logic.

Reply:
Totally Okay.